WebDevStudios/wd_s
# Removes lazy load filters

Closes #588

### DESCRIPTION
Removes the lazy load filters.

Previously, we were adding `loading="lazy"` to all images and filtering the output of the core image block to add lazy load.

With [WP 5.5](https://make.wordpress.org/core/2020/07/14/lazy-loading-images-in-5-5/), lazy load is the default behavior so we no longer need to add lazy loading with these filters.

### OTHER


### STEPS TO VERIFY
1. Check out the branch
2. Add an image to a page or post
3. Add an image gallery to a page or post
4. Inspect the images to see that the lazy loading tag is added by default

### DOCUMENTATION
Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

Yes, we'll want to remove references to the lazy loading filters.
